app/views/course/experience_points_disbursement/new.slim

```diff
@@ -33,12 +33,22 @@
       tr
         th = CourseUser.human_attribute_name(:name)
         th = Course::ExperiencePointsRecord.human_attribute_name(:points_awarded)
+        th
     tbody
+      - display_copy_button = @experience_points_disbursement.experience_points_records.length > 1
       = f.simple_fields_for :experience_points_records do |record_fields|
-        = content_tag_for(:tr, record_fields.object.course_user)
-          td = display_course_user(record_fields.object.course_user)
-          td = record_fields.input :points_awarded, label: false,
-                                   input_html: { class: 'points_awarded' }
+        - course_user = record_fields.object.course_user
+        = content_tag_for(:tr, course_user)
+          td
+            = link_to display_course_user(course_user),
+                      course_user_experience_points_records_path(current_course, course_user)
+          td
+            = record_fields.input :points_awarded, label: false,
+                                                   input_html: { class: 'points_awarded' }
+          td
+            - if display_copy_button
+              - display_copy_button = false
```

Comment:
I think he only want to show this field for the first row. 

Actually you can try `index` with enumerable instead, check out [this stackoverflow link](http://stackoverflow.com/questions/4853373/rails-fields-for-with-index):